src/content/en/2021/performance.md

```diff
@@ -1,20 +1,532 @@
+
+In both cases, images comprised the majority of LCP elements. This warrants a deeper dive into how those images are loading.
+
+{{ figure_markup(
+  image="performance-lcp-antipatterns.png",
+  caption="LCP elements with performance anti-patterns",
+  description="Bar chart showing 9.3% of pages on mobile use native lazy load on the LCP element, 16.5% probably lazy load, and 0.4% use async decode.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQj6-t49BNV_5W-w83AABkUoo6klfyUyIz13yKShLAzK8qGs5lJ9TKcggIEp6JgxikVF-UJBAHpsrNl/pubchart?oid=130632749&format=interactive",
+  sheets_gid="730149797",
+  sql_file="lcp_element_data.sql"
+  )
+}}
+
+For user experience, we want LCP elements to load as fast as possible. User experience is why LCP was selected as one of the Core Web Vitals. We do not want it to be lazy-loaded nor decoded asynchronously as that further delays the render. However, we can see that 9.3% of pages used the native loading=lazy flag on the LCP `<img>` element.
+
+Not all browsers support native lazy loading. Popular lazy loading polyfills detect a "lazyload" class on an image element. Thus, we can identify more possibly lazy-loaded images by adding images with a "lazyload" class to the total. The percent of sites probably lazy loading their LCP `<img>` element jumps up to 16.5% on mobile.
+
+Lazy-loading your LCP element will result in worse performance. Don't do it! WordPress was an early adopter of native lazy loading. The early method was a naive solution applying lazy loading to all images, and the results showed a <a hreflang="en" href="https://web.dev/lcp-lazy-loading/">negative performance correlation</a>. They were able to use this data to implement a more nuanced approach for better performance.
+
+Luckily, only 0.4% of sites used the async decode directive for their LCP image. And, the negative impact of asynchronous decode is not as high as lazy loading.
+
+{{ figure_markup(
+  content="354",
+  caption="Websites attempted to use native lazy-loading on LCP elements that are not images or iframes",
+  classes="big-number",
+  sheets_gid="1423728540",
+  sql_file="lcp_element_data.sql"
+  )
+}}
+
+Interestingly, 354 origins on desktop attempted to use native-lazy loading on HTML elements that do not support the loading attribute (e.g., `<div>`). The loading attribute is only supported on `<img>` and  sometimes `<iframe>` elements (see <a hreflang="en" href="https://caniuse.com/loading-lazy-attr">caniuse</a>).
+
+### Cumulative Layout Shift (CLS)
+
+{{ figure_markup(
+  image="performance-CLS-by-device.png",
+  caption="CLS performance by device",
+  description="Stacked bar chart showing CLS performance for desktop and mobile devices. For desktop devices, 62% of websites had good (< 0.1), 23% needs improvement, and 15% poor (>= 0.25) CLS performance. For mobile, 62% were good, 21% needs improvement, and 17% poor.",
```

Comment:
inconsistent between `good` and `poor`

Reply:
I'm confused what you mean by "inconsistent"? Are the numbers wrong?

Reply:
Not sure what she means by that either, but did spot something else in this description: "good (< 0.1)....poor (>= 0.25)". all CWV borderlines were clarified as to be in the better category a while back. So my understand is good is <= 0.1 and poor is > 0.25.

https://web.dev/vitals/

![image](https://user-images.githubusercontent.com/10931297/141685951-9a300539-7cb1-4aff-b79b-d9938f59aa7c.png)

Reply:
@tunetheweb I think it has to do with verb usage... see later comment I made

Reply:
I was just clarifying that the point I was about to make, was a completely separate point (but one that should be addressed).

Reply:
OH. I did not make the charts and assumed they were all correct. If that's the case then we'll need to redo them all 😭

Reply:
Per rick "The charts are correct when we use CrUX BQ as the data source. The underlying histograms use < not <= for the upper limit of each bin." Marking as resolved.